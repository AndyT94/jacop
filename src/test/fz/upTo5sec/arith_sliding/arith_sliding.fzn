var 4..4: z:: output_var;
array [1..7] of var int: x:: output_array([1..7]) = [0,0,1,2,0,0,-3];
solve  satisfy;
