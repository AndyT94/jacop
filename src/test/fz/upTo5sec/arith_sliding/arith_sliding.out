z = 4;
x = array1d(1..7, [0, 0, 1, 2, 0, 0, -3]);
----------